controllers/reconcilers/kserve_serverless_inferenceservice_reconciler.go

```diff
@@ -38,8 +38,6 @@ func NewKServeServerlessInferenceServiceReconciler(client client.Client, clientR
 	subResourceReconciler := []SubResourceReconciler{
 		NewKserveServiceMeshMemberReconciler(client),
 		NewKserveRouteReconciler(client),
-		NewKServeMetricsServiceReconciler(client),
-		NewKServeMetricsServiceMonitorReconciler(client),
```

Comment:
Would this break existing metrics functionality for serverless?

Reply:
I think I got my answer: https://github.com/opendatahub-io/odh-model-controller/pull/250#discussion_r1705967262

Reply:
The old metrics reconcilers for kserve serverless were present pre 2.8 (EUS release)
Kserve Raw support will happen after 2.13 at minimum (2.12 is a EUS if i'm not wrong)
I can keep an "empty reconciliation" loop around, but the old resources should long by the time kserve raw is GA. This was my reasoning for assuming that it's safe to delete the old reconcilers for serverless

Reply:
the upgrade path for any install that still has those reconcilers should look like 
`2.7 or lower` -> `2.8` -> `2.12` (at minimum) -> kserve raw GA. 
If i'm mistaken in this assumption I'll re-add the cleanup.